src/rgw/rgw_common.h

```diff
@@ -1384,7 +1384,7 @@ static inline int hexdigit(char c)
   return -EINVAL;
 }
 
-static inline int hex_to_buf(const char *hex, char *buf, int len)
+static inline int hex_to_buf(const char *hex, unsigned char *buf, int len)
```

Comment:
Seems valid, though I think treating char as signed is harmless in this sort of (C) code

Reply:
yes it is infact harmless . just matters compile time